test/test_files/common/okwitheof.test

```diff
@@ -0,0 +1,9 @@
+-DATASET CSV tinysnb
+
+--
+
+-CASE OkWithEOFTest
+
+
+-STATEMENT CALL show_tables() RETURN *
+---- ok
```

Comment:
Instead of adding a new test file, i would remove the newline from an existing test file.

Reply:
I dont think that we have any tests which end with ----ok, I could put it inside of comments.test so that we don't have to create a new testfile